tokenLists/neutron.json

```diff
@@ -360,5 +360,12 @@
     "token": "ibc/B4C0F9BCB087D78ED8660D2824C4B81682840D0B69E83C62A78CA5874DD742A8",
     "icon": "/img/autism.jpg",
     "decimals": 6
+  },
+  {
+    "protocol": "Chihuahuawifhat",
+    "symbol": "BADDOG",
+    "token": "ibc/2FFE07C4B4EFC0DDA099A16C6AF3C9CCA653CC56077E87217A585D48794B0BC7",
```

Comment:
ibc denom doesn't look right for all chains except osmosis

eg: on neutron this seems to be the correct one `ibc/EC279EDDCD229637600C1027C1F1074A1EA09B5B30013D00C10DAB8572DB0B4B`